src/main/java/com/github/adamyork/wiremock/transformer/VelocityResponseTransformer.java

```diff
@@ -134,7 +134,7 @@ private Context addBodyToContext(final String body, final Context context) {
                 .orElseGet(() -> {
                     final JSONParser parser = new JSONParser(JSONParser.MODE_JSON_SIMPLE);
                     return Unchecked.supplier(() -> {
-                        final JSONObject json = (JSONObject) parser.parse(body);
+                        final Object json = parser.parse(body);
```

Comment:
I don't think down casting to the most primitive type in java is what we want to do here. You are basically disregarding type safety. There are really only two valid wrapper types per the JSON rfc; [] (JSONArray) and {} (JSONObject). There may an interface we can assign the cast resulted of parse to instead.

Reply:
But parse method declared as 
`public Object parse(String in) throws ParseException`
and method`s return type is Object. So I offer don`t cast to any other type.
Also I had test this variant and all works fine.

Reply:
documentation says it returns a JSONElement. https://static.javadoc.io/com.google.code.gson/gson/2.6.2/com/google/gson/JsonElement.html

Reply:
hi , please try the latest release : https://github.com/adamyork/wiremock-velocity-transformer/releases/tag/2.3-release , and let me know if this fixes your issue.

Reply:
all works fine, thanks!